subprojects/packagefiles/orocos-kdl/subprojects/eigen.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = eigen-3.4.0
+source_url = https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.bz2
+source_filename = eigen-3.4.0.tar.bz2
+source_hash = b4c198460eba6f28d34894e3a5710998818515104d6e74e5cc331ce31e46e626
+patch_directory = eigen
+
+[provide]
+eigen3 = eigen_dep
+
```

Comment:
Not necessary empty line.

Reply:
The sanity checks are actually complaining about this file, am I not allowed to include it? I'd like whatever project I include orcos-kdl in to automatically download eigen3 if it doesn't already exist on the system